eva_submission/eload_utils.py

```diff
@@ -162,3 +162,16 @@ def get_hold_date_from_ena(project_accession, project_alias=None):
         if not hold_date:
             raise ValueError(f"Couldn't get hold date from ENA for {project_accession} ({project_alias})")
     return hold_date
+
+
+def backup_file(file_name):
+    suffix = 1
+    backup_name = file_name + '.' + str(suffix)
```

Comment:
also bit of a nitpick but if you do `f'{filename}.{suffix}'` you don't need the string casts everywhere...

Reply:
Done